docker/entrypoint.sh

```diff
@@ -27,15 +27,32 @@ if [[ "$1" = "/opt/bitnami/scripts/magento/run.sh" || "$1" = "/opt/bitnami/scrip
     info "** Magento setup finished! **"
 fi
 
-info "**update magento"
 
-#TODO: do it only on Admin ?
-cd /bitnami/magento
-bin/magento config:set web/secure/base_url https://$MAGENTO_HOST/
-bin/magento config:set web/unsecure/base_url http://$MAGENTO_HOST/
-php bin/magento setup:upgrade && \
-php bin/magento setup:static-content:deploy -f && \
-php bin/magento cache:flush
 
+if [[ "$MAGENTO_USE_EFS" = "yes" && -d /bitnami/magento/bin ]]; then
+
+    info "**update magento credentials"
+    #TODO: do it only on Admin ?
+    cd /bitnami/magento
+    mkdir -p /bitnami/magento/var/composer_home/
+    cat <<END > /bitnami/magento/var/composer_home/auth.json
+{
+    "http-basic": {
+        "repo.magento.com": {
+            "username": "$MAGENTO_MARKETPLACE_PUBLIC_KEY",
```

Comment:
TODO - We need to have an instruction late on populating this as part of the readme

Reply:
I added them in the **Project Pre-requisites** section of the README